axonserver-grpc/pom.xml

```diff
@@ -9,6 +10,7 @@
 
     <artifactId>axonserver-grpc</artifactId>
     <name>AxonServer - GRPC proto generated code</name>
+    <packaging>bundle</packaging>
```

Comment:
Why does this package need to be a bundle?
These classes should be available to bundles, but they should be exported by the 'system bundle', right?